automated-comments/go/general/section_about_comments.md

```diff
@@ -0,0 +1,27 @@
+
+---
```

Comment:
Is there a reason for this whitespace and `---` here?

Reply:
That was to separate it from the other comments in exalysis. Will remove it as this will probably be done differently with the automentoring.

Reply:
Brilliant. Thanks :)